cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs

```diff
@@ -595,6 +595,12 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl pallet_broker_runtime_api::BrokerApi<Block, Balance> for Runtime {
+		fn sale_price() -> Option<Balance> {
```

Comment:
And do we maybe want to expose more stuff?

Reply:
It would be useful to fetch the unclaimed revenue of a region contributed to the instantaneous pool. I can add that as well.

Reply:
> It would be useful to fetch the unclaimed revenue of a region contributed to the instantaneous pool.

After thinking about this a bit more not sure if adding this would make sense. Getting the unclaimed revenue can require quite a lot of reading(reading `InstaPoolHistory` per each timeslice). So having this exposed could be an easy spam vector.